snuba/datasets/__init__.py

```diff
@@ -82,6 +83,12 @@ def default_conditions(self):
         """
         return []
 
+    def get_extensions_conditions(
```

Comment:
This is probably better as a code comment than a review comment just in case this ends up sticking around longer than expected. (I know that never happens, but…)

Reply:
_(I know that never happens, but…)_

Yeah, I never saw something like that happening either :D

Will add the comment.